### ISSUE TITLE
WebDockableFrame doesn't show AWT components

### ISSUE THREAD

**Issue by Husky-s** [mgarin/weblaf#528]
WebDockableFrame doesn't show AWT elements like Label and GLCanvas (from JOGL)

**Comment by mgarin** [comment]
I'll test this tomorrow, but for now -
Was there anything special with the placement of AWT elements within `WebDockableFrame`?
Or you simply put it straight as the content?

**Comment by Husky-s** [comment]
I put a webpanel as content in which there were awt components.

**Comment by mgarin** [comment]
Thanks, i'll try it.

**Comment by mgarin** [comment]
I found the problem and it seem to be consistent with all AWT components - they cannot be displayed under any other Swing component, transparent or not. Which in case of the `WebDockablePane` is a transparent glass pane (`DockablePaneGlassLayer`) added within `WebDockablePane` to track mouse location and support resize operations for the frames.

I usually do not have any hidden glass layers on components, but in case of `WebDockablePane` it is necessary to consistently support resize operation, even on top other components within frames and content area that might be catching mouse events.

Unfortunately I haven't found any possible solution for that issue with AWT components, will see if I can find anything on it.

**Comment by mgarin** [comment]
Well, I managed to find a wild workaround that Swing uses for its own glass pane. It's not really a public API so I've added new method to `ProprietaryUtils` - `enableMixingCutoutShape ( Component )`. It allows mixing lightweight and heavyweight components on the same layer. There will still be some visual glitches if anything is drawn on top of an AWT component, but it will properly display within `WebDockablePane` now and generally should work as expected. 

I can't really do much about Swing components intersecting with AWT ones (for instance a tooltip or a menu) - they will still hide parts of AWT components while positioned on top of them, but AWT components will not be permanently hidden anymore.

New v1.2.10 snapshot build will soon be available with the added changes and the final v1.2.10 update will most probably be available this Friday or Saturday.
